src/services/saml-config-service.js

```diff
@@ -71,7 +71,7 @@ export default class SamlConfigService {
         this._identityProviderMap.set(provider.key, {
           ...provider,
           resolvedMetadata,
-          lastUpdatedOn: new Date()
+          lastUpdateOn: new Date()
```

Comment:
Oh we had a typo here? 😳

Reply:
:scream: That happened accidenatlly. Thanks for pointing it out. I reverted it!

Reply:
👌